### DISCUSSION TITLE
Process window not showing git console output (local user)

### DISCUSSION THREAD

**Discussion by oryz** [gitextensions/gitextensions#10126] [Q&A]
I have a weird symptom on my machine, started a couple of days ago (corporate laptop so might have received some updates over the weekend) where when I execute any git command the Process dialog that pop up is not showing the text of the console like it normally would (see attachment). The command does execute and I can see it in the Git Command Log window afterwards.

https://user-images.githubusercontent.com/20960028/182526764-d131650e-2dac-4952-9aba-3567dc7850aa.mp4

What I've tried:
- uninstalling and reinstalling
- run as local administrator

Re-installation did not not help, but running as a local admin has solved the issue. But the question is why? I don't see why elevated permissions would have anything to do with it (the clue I was going by was - as can be seen in the video - the first line displayed in the window before it goes blank is "Starting root process").

Does anyone have any ideas/clues/suspicions as to what might be causing such an issue? I'm completely flabbergasted by this one!

Cheers!

**Comment by RussKie** [comment]
Judging by the background colour the console emulator is being launched.

Do you have all these files in C:\Program Files (x86)\GitExtensions\ConEmu?
```
08/03/2022  08:58 PM             2,245 CmdInit.cmd
18/04/2022  08:49 PM         2,263,952 ConEmu.exe
18/04/2022  08:49 PM         2,548,624 ConEmu64.exe
18/04/2022  08:49 PM           344,976 ConEmuC.exe
18/04/2022  08:49 PM           373,136 ConEmuC64.exe
18/04/2022  08:49 PM           530,320 ConEmuCD.dll
18/04/2022  08:49 PM           626,064 ConEmuCD64.dll
18/04/2022  08:49 PM           378,256 ConEmuHk.dll
18/04/2022  08:49 PM           459,152 ConEmuHk64.dll
13/04/2020  01:20 PM             6,044 GitShowBranch.cmd
              10 File(s)      7,532,769 bytes
```

Are any of the dlls blocked by Windows or AV?

. . . 

As a (temporary?) workaround you could disable the console emulator (though it degrades the UX): https://git-extensions-documentation.readthedocs.io/en/master/settings.html#git-extensions-advanced-general-use-console-emulator-for-console-output-in-command-dialogs
![image](https://user-images.githubusercontent.com/4403806/182758909-8c50e5be-c369-44e7-beeb-05ffab4acec4.png)

**Comment by oryz** [comment]
Thanks for the reply. 

Yes all the files are there. 

I'm not sure if the AV is blocking anything unfortunately its quite black box, but can raise with the organisation. 

Disabling the Console Emulator works!
![image](https://user-images.githubusercontent.com/20960028/182800446-7296f829-eecc-4d5d-8dd9-45e100e84cd1.png)

**Reply by RussKie** [reply]
Are you able to execute ConEmu64.exe?
![image](https://user-images.githubusercontent.com/4403806/182985908-5fb4a3ae-ed72-4530-a2f4-ee0360fa721c.png)

**Comment by oryz** [comment]
> Are you able to execute ConEmu64.exe?

yes.
